### ISSUE TITLE
API Token deletion throws Router error

### ISSUE THREAD

**Issue by danieltorrer** [laravel/jetstream#340]
- Jetstream Version: 1.4.0
- Jetstream Stack: Inertia
- Laravel Version: 8.0.0
- PHP Version: 7.3
- Database Driver & Version:
MySQL (where do I see my driver version?)
sqlite

### Description:
The following error is thrown when trying to delete an API Token
```
[Vue warn]: Error in v-on handler: "TypeError: Cannot read property 'hasOwnProperty' of null"

found in

---> <ApiTokenManager> at resources/js/Pages/API/ApiTokenManager.vue
       <AppLayout> at resources/js/Layouts/AppLayout.vue
         <Index> at resources/js/Pages/API/Index.vue
           <Inertia>
             <Root>
```
```
TypeError: Cannot read property 'hasOwnProperty' of null
    at Router.normalizeParams (dashboard:1079)
    at new Router (dashboard:1063)
    at route (dashboard:1261)
    at VueComponent.deleteApiToken (app.js:3357)
    at click (app.js:26382)
    at invokeWithErrorHandling (app.js:29752)
    at HTMLButtonElement.invoker (app.js:30077)
    at HTMLButtonElement.original._wrapper (app.js:35436)
```

### Steps To Reproduce:

1- Log in
2- Go to API Tokens menu
3- Create any API Token
4- Try to delete it under the section "Manage API Tokens"

Tried with both MySQL dirver and sqlite. Both drivers throw the error.

**Comment by cdietrich67** [comment]
Hi,

I had the same problem, you need to edit your vue file :
resources/js/Pages/API/ApiTokenManager.vue

At the deleteApiToken() method, change this line

`this.deleteApiTokenForm.delete(route('api-tokens.destroy', this.managingPermissionsFor), {`

to

`this.deleteApiTokenForm.delete(route('api-tokens.destroy', this.apiTokenBeingDeleted), {`

It worked for me

**Comment by bonzai** [comment]
Fixed in 1.4.1 (afc89ad834d179dd39a0b86815b48dac8f039f69)
